jupyterlab_git/git.py

```diff
@@ -110,6 +110,13 @@ class Git:
     def __init__(self, contents_manager):
         self.contents_manager = contents_manager
         self.root_dir = os.path.expanduser(contents_manager.root_dir)
+        
+    def strip_and_split(self, s):
```

Comment:
```suggestion
    @staticmethod
    def strip_and_split(s):
```

As the object is not needed, it is better to define this as a static method or to place it outside the class as utility function

Reply:
Great point thanks. I moved the function outside the object.